src/components/auth/RedirectLogout.tsx

```diff
@@ -0,0 +1,12 @@
+import React, {useEffect} from "react";
+import {logoutHelper} from "../../actions/user";
+
+export const RedirectLogout = (): JSX.Element => {
+	useEffect(() => {
+		logoutHelper();
+	}, []);
+
+	return (
+		<div>Logged Out!</div>
```

Comment:
Add banner and style with a bit more information. For example. "You have logged out. To log back in click here."